### PR TITLE
Add tutorial: GC MS data processing [metabolomics]

### PR DESCRIPTION
**Pull request by xtrojak**
This PR adds a new tutorial to the _metabolomics_ category. The tutorial is focusing on explanation of GC-MS data processing using tools [XCMS](https://github.com/sneumann/xcms), [RAMClustR](https://github.com/cbroeckl/RAMClustR), [RIAssigner](https://github.com/RECETOX/RIAssigner), and [matchms](https://github.com/matchms/matchms).

### CODE REVIEW

File: Gemfile.lock
Comment (hexylena) [inline Gemfile.lock:1]: could you please revert this change, this file must be kept.

File: Gemfile
Comment (hexylena) [inline Gemfile:1]: could you please revert this change, this file must be kept.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+>    > <comment-title> Minutes vs. seconds </comment-title>
+>    >
+>    > You might notice that in the last **XCMS** step, we converted the retention times to minutes. However, in this step, we are using seconds again. The reason is that **RAMClustR** converted them internally again to seconds. Regarding the reference compound list, this database already has its retention times in seconds.
+>    {: .comment}
+>
+>    > <details-title> Kováts method </details-title>
+>    >
+>    > The Kováts retention index (or Kováts index) of a compound is its retention time normalized to the retention times of adjacently eluting n-alkanes. It is based on the fact that the logarithm of the retention time converges to the number of carbon atoms in the alkane. For an isothermal chromatogram, you use the following equation to calculate the Kováts index: 
+>    > 
+>    > $$I = 100 \times \frac{\mathtt{log}~t_x - \mathtt{log}~t_n}{\mathtt{log}~t_{n+1} - \mathtt{log}~t_n}$$
```
Comment (xtrojak) [inline topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:367]: I have encountered a problem with rendering this mathematical expression. It occurs when used in a line block inside the details block of the hands-on section. The math expression is displayed before the details block is expanded.

Collapsed details block:
collapsed

Expanded details block:
expanded

I tried to find something similar in previous materials, it seems like [here](https://github.com/galaxyproject/training-material/blob/d13771e996e13fdc821a28556692a8e4e1675291/topics/single-cell/tutorials/scrna-raceid/tutorial.md?plain=1#L429) someone had the same issue. But there, it is rendering fine, so maybe this only occurs when running the GTN page in Gitpod?

File: Gemfile.lock
Comment (xtrojak) [reply Gemfile.lock:1]: Ops, my bad, done in [2b727de](https://github.com/galaxyproject/training-material/pull/3949/commits/2b727de549698ca8dcefc634d60af3aa5904c086).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:367]: Ahhh interesting. What an odd corner case. Ok, I will look into it. Thanks for alerting us!

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+>    {: .comment}
+>
+{: .hands_on}
+
+> <details-title> Overview of the spectral similarity scores </details-title>
+> > <h5>Cosine Greedy</h5>
+> > The cosine score, also known as the dot product, is based on representing the similarity of two spectra through the cosine of an angle between the vectors that the spectra produce. Two peaks are considered as matching if their *m/z* values lie within the given tolerance. Cosine greedy looks up matching peaks in a "greedy" way, which does not always lead to the most optimal alignments.
+> >
+> > This score was among the first to be used for looking up matching spectra in spectral libraries and, to this day, remains one of the most popular scoring methods for both library matching and molecular networking workflows.
+> > <p>&nbsp;</p>
```
Comment (hexylena) [inline topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:425]: this should not be necessary

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+>
+> > <h5>Cosine Hungarian</h5>
+> > This method computes the similarities in the same way as the *Cosine Greedy* but with a difference in *m/z* peak alignment. The difference lies in that the Hungarian algorithm is used here to find matching peaks. This leads to the best peak pairs match but can take significantly longer than the "greedy" algorithm.
+> > <p>&nbsp;</p>
```
Comment (hexylena) [inline topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:429]:
```suggestion
> ### Cosine Hungarian
> This method computes the similarities in the same way as the *Cosine Greedy* but with a difference in *m/z* peak alignment. The difference lies in that the Hungarian algorithm is used here to find matching peaks. This leads to the best peak pairs match but can take significantly longer than the "greedy" algorithm.
>
```

these all seem to have an extra level of indenting than needed

**Review by hexylena — COMMENTED** [review]
Looks good from the formatting side, just a few notes.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+- This processing workflow is complementary to CAMERA and metaMS.
+requirements :
+  - type: "internal"
+    topic_name: metabolomics
+    tutorials: 
+      - lcms-preprocessing
+contributors:
+- xtrojak
+- hechth
+- maximskorik
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: Cool, thanks, done in [2e14a10](https://github.com/galaxyproject/training-material/pull/3949/commits/2e14a1058fde922a9678c1761b92a098529f248a).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md (hunk 2 above)
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: Yeah, I don't know how it got there, thanks for noticing. Fixed in [fcd889f](https://github.com/galaxyproject/training-material/pull/3949/commits/fcd889fc380f44e62dc43e45bcd5e3150870903e).

**Review by hexylena — DISMISSED** [review]
Thanks for all the formatting fixes @xtrojak, looks great from my side.

**Comment by hexylena** [comment]
Pinging @melpetera @lecorguille @foellmelanie if any of you could look at it as metabolomics maintainers :)

**Comment by melpetera** [comment]
Hi,
Thank you for providing this new training material. 
If by next wednesday nobody have done it yet, I will try to have a look at the content from a "metabolomic user" point of view (although I am no expert in GC-MS). Sorry for the delay.

**Comment by melpetera** [comment]
Hi there,
I just finished to try this tutorial. Great job there :)
I have a few comments and suggestions. I am not used to reviewing on GitHub, so I will try to use the dedicated functionnalities, but I appology in advance in case I do not use the system correctly. 
I will do this tomorrow morning, sorry for the delay.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga:1]: It is strongly preferred that the workflow match the training material as closely as possible so we can test the tutorial more easily via the workflows.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,482 @@
...
+>
+>    ```
+>    https://zenodo.org/record/7701862/files/8_qc_no_dil_milliq.raw
+>    https://zenodo.org/record/7701862/files/21_qc_no_dil_milliq.raw
+>    https://zenodo.org/record/7701862/files/29_qc_no_dil_milliq.raw
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md collection=true format="mzml" collection_name="input" renaming=false %}
+>
+> 3. Make sure your data is in a **collection**. You can always manually create the collection from separate files:
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:81]: That's a good point, I have added a comment on that in [43f0a69](https://github.com/galaxyproject/training-material/pull/3949/commits/43f0a69690f709b2da635b4eac63c3fcc2a505e9).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,482 @@
...
+>
+>    {% snippet faqs/galaxy/collections_build_list.md %}
+>
+> 4. Import the following extra files from [Zenodo]({{ page.zenodo_link }}):
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:87]: I added a comment on this and also relevant tip snippet (I decided to use a more general tip on datatypes, not just for change of format) in [7650cef](https://github.com/galaxyproject/training-material/pull/3949/commits/7650cefb4557786b3921503c65355f15132437cc).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,482 @@
...
+>
+>    > <comment-title> Output - `input.raw.RData` </comment-title>
+>    >
+>    > Collection of `rdata.msnbase.raw` files. `Rdata` file that is necessary in the next step of the workflow. These serve for an internal R representation of **XCMS** objects needed in the further steps.
+>    {: .comment}
+{: .hands_on}
+
+# Peak detection using XCMS
+
+The first step in the workflow is to detect the peaks in our data using **XCMS**. This part, however, is covered by a [separate tutorial]({{ site.baseurl }}/topics/metabolomics/tutorials/lcms-preprocessing/tutorial.html). Although the tutorial is dedicated to LC-MS data, it can also be followed for our GC-MS data. Therefore, in this section, we do not explain this part of the workflow in detail but rather refer the reader to the dedicated tutorial. Please also pay attention to the parameter values for individual Galaxy tools, as these can differ from the referred tutorial and are adjusted to our dataset.
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:157]: I'm not sure what you mean by "without waiting for the results". Do you mean we could provide the data already processed by XCMS and allow the trainee to upload them and effectively skip XCMS part? Or did you have something else in mind?

@hechth what is your opinion?

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,482 @@
...
+
+> <details-title> RAMClust method </details-title>
+> 
+> The RAMclust similarity scoring utilises a Gaussian function, allowing flexibility in tuning correlational and retention time similarity decay rates independently, based on the dataset and the acquisition instrumentation. The correlational relationship between two features can be described by either MS-MS, MS-idMS/MS, or idMS/MS-idMS/MS values, and we support several correlation methods (e.g. Pearson’s method) to calculate similarity. These are parametrised by `sigma_t` and `sigma_r`, Gaussian tuning parameters of retention time similarity and correlational score, respectively, between feature pairs.
+> 
+> Similarities are then converted to dissimilarities for clustering. The similarity matrix is then clustered using average or complete linkage hierarchical clustering. The dendrogram is cut using the `cutreeDynamicTree` function from the package `dynamicTreeCut`. For this application, the minimum module size is set to 2, dictating that only clusters with two or more features are returned, as singletons are impossible to interpret intelligently.
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: You are right, I corrected this in [e95a233](https://github.com/galaxyproject/training-material/pull/3949/commits/e95a2339b127d0c0b1c0bedbc89beb06e4a3ee13).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 9]
```diff
@@ -0,0 +1,482 @@
...
+>        - *"Minimal cluster size"*: `5`
+>        - *"Maximal tree height"*: `0.9`
+>
+>    You can leave the other parameters with their default values.
+>
+{: .hands_on}
+
+The spectral data comes as an `.msp` file, which is a text file structured according to the **NIST MSSearch** spectra format. `.msp` is one of the generally accepted formats for mass spectral libraries (or collections of unidentified spectra, so called spectral archives), and it is compatible with lots of spectra processing programmes (MS-DIAL, NIST MS Search, AMDIS, etc.). Because `.msp` files are text-based, they can be viewed as simple `txt` files. You can use any text editor that you have on your computer or use Galaxy's built-in editor. In this tutorial, we use the Galaxy editor to check the contents of the file:
+
+> <hands-on-title> Data Exploration </hands-on-title>
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:301]: There is no particular reason to use Editor in this situation. Based on your suggestion, I simplified the data exploration to us View data (eye icon) in [74af30f](https://github.com/galaxyproject/training-material/pull/3949/commits/74af30fea5a131a3574351bfee214f619cdafdea).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,482 @@
...
+>    >
+>    {: .details}
+>
+{: .hands_on}
+
+`.msp` files can contain one or more mass spectra, these are split by an empty line. The individual spectra essentially consist of two sections: metadata and peaks. The metadata consists of compound name, spectrum type (which is centroid in this case), ion mode, retention time, and the number of *m/z* peaks. If the compound has not been identified, as in our case, the **NAME** can be any arbitrary string. It is best, however, if that string is unique within that `.msp` file. The metadata fields are usually unordered, so it is quite common for one `.msp` file to contain **NAME** as the first metadata key and for another `.msp` to have this key somewhere in the middle. The keys themselves also aren't rigid and can have different names (e.g., **compound_name** instead of **NAME**) or store information not present in this `.msp`, such as the ionization mode. The `msp` format is inherently not standardized and various flavours exist.
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: I simplified this comment in [2887509](https://github.com/galaxyproject/training-material/pull/3949/commits/2887509835c7f4cf5b4075508a0d181bcc978462).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 11]
```diff
@@ -0,0 +1,482 @@
...
+To have more information available for compound identification, we computed the retention index using **RIAssigner** and added it to our deconvoluted spectra.
+We then finally compare the deconvoluted spectra against the reference library using **matchms**, using a retention index threshold to pre-filter our matches and then computing the cosine similarity for the spectra within the retention index tolerance.
+
+This workflow leverages the correlation of features across samples to improve the annotation performance, trying to capture even low abundant signals.
+It is suitable for both high- and low-resolution GC-MS data acquired in either profile or centroid mode by adapting the individual tool parameters or in-/excluding individual steps which are specific to a certain data format.
+
+Before further statistical analysis, the information from the scores excel sheet can be included in the library of deconvoluted spectra manually to add the putative identifications. The further processing can then leverage this information and the intensity information contained in the **SpecAbundance** output from **RAMclustR**.
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: You are right, `scores table` is more suitable in this context ([3c9622e](https://github.com/galaxyproject/training-material/pull/3949/commits/3c9622e1cfcbfcde9885bec35af9ff337be0195b)).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 12]
```diff
@@ -0,0 +1,482 @@
...
+
+A deviation in retention time occurs from one sample to another, especially when you inject large sequences of samples. This step aims at correcting retention time drift for each peak among samples.
+
+> <hands-on-title> Retention time correction </hands-on-title>
+>
+>  {% tool [xcms adjustRtime (retcor)](toolshed.g2.bx.psu.edu/repos/lecorguille/xcms_retcor/abims_xcms_retcor/3.12.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"RData file"*: `xset.merged.groupChromPeaks.RData` (output of **xcms groupChromPeaks (group)** {% icon tool %})
+>    - *"Method to use for retention time correction"*: `PeakGroups - retention time correction based on aligment of features (peak groups) present in most/all samples.`
+>        - *"Minimum required fraction of samples in which peaks for the peak group were identified"*: `0.7`
+>        - *"Smooth method"*: `loess - non-linear alignment`
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: There is not reason for that. I noticed generating tutorial file automatically from the workflow file leaves some default parameters there. I already removed several of them before. Thanks for noticing, I removed this line in [3907ed5](https://github.com/galaxyproject/training-material/pull/3949/commits/3907ed515132d54c6d29a2dcbc9463b97ca598d8).

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 13]
```diff
@@ -0,0 +1,482 @@
...
+>
+>  {% tool [xcms groupChromPeaks (group)](toolshed.g2.bx.psu.edu/repos/lecorguille/xcms_group/abims_xcms_group/3.12.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"RData file"*: `xset.merged.groupChromPeaks.adjustRtime.RData` (output of **xcms adjustRtime (retcor)** {% icon tool %})
+>    - *"Method to use for grouping"*: `PeakDensity - peak grouping based on time dimension peak densities`
+>        - *"Bandwidth"*: `3.0`
+>        - *"Minimum fraction of samples"*: `0.9`
+>        - *"Width of overlapping m/z slices"*: `0.01`
+>    - *"Get the Peak List"*: `Yes`
+>        - *"Convert retention time (seconds) into minutes"*: `Yes`
+>        - *"If NA values remain, replace them by 0 in the dataMatrix"*: `Yes`
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md]: You are completely right, I changed the value to `'No'`.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga:1]: Some dependency issues emerged in the `0.3.2+galaxy1`. Those we pinned and new release `0.3.4+galaxy1` was made. I changed `RIAssigner` version in the workflow ([a726fb0](https://github.com/galaxyproject/training-material/pull/3949/commits/a726fb0c2af4a24843f6a50c6f522a629b43f6dd)).

**Comment by xtrojak** [comment]
> Thank you for this new tutorial! Reviewing this was the opportunity to test the corresponding tools and get an overview of the outputs :) I made some suggestions this morning, I hope I did it the right way (it seems to be ok, at least it looks similar to the reviews I have seen on GitHub!).

@melpetera Thank you for a very detailed review! I tried to cover all the issues you discovered. Let me know if more needs to be done.

**Comment by hechth** [comment]
@hexylena and @melpetera thank you very much for the review and spending the time on improving the training! Your feedback is highly appreciated :).

**Review by hexylena — DISMISSED** [review]
If @melpetera approves then let's merge!

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (melpetera) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:115]: Should be ok this way. If you get feedback about this in the future anyway, you can always add a comment in the FAQ section later.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (melpetera) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:185]: About this point, the idea that I did not expressed well was that since the XCMS processing part can be a bit long to run, and since you refered to the LC-MS XCMS tutorial, it may have been interesting to warn the trainee about the time needed to run the following steps and thus to invite him/her not to wait to launch the XCMS part of the GC-MS tutorial (that can be launch all in a raw) even if he/she plans to take a look ad the LC-MS tutorial (the idea was more to optimise the learning time with a "reading while it runs" approach). Sorry for the potential misunderstanding.

Anyway, it is a good idea to provide a final XCMS object to save time so thank you for your modifications. 
Still, I am not sure of the place where to add the "Skip" part in the tutorial: it is currently at the end of the introduction (before the agenda) when I would have expected it to be just before the time-consuming steps (the XCMS steps in fact, the conversion from raw to mzML being sufficiently fast). Moreover, the convertion part and data format issue is an important point (not covered in the LC-MS XCMS tutorial), so keeping it as mandatory would be a good idea. In addition, in the "Import the data into Galaxy" there are some precious comments about data (e.g. the comment about the extra files), so it would be a pity to enable trainees to skip it just because of computing time. 

In conclusion, I would suggest to move the "Skip" part from introduction to the "Peak detection using XCMS" part (between the first paragraph and the "Peak picking" part), and to reduce its content to the "XCMS" object only, mentioning explicitly that this corresponds to the final dataset from the XCMS step (going from "Peak picking" to "Integrating areas of missing peaks") that can be used to save time if processing takes too long and/or if the trainee already know the XCMS steps and wants to proceed with the GC-specific parts. 

I hope this time I am at least a bit clearer than my previous suggestion but that is not easy to explain. So if needed, do not hesitate to contact me by e-mail and then we can plan a short call to discuss it.

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md [hunk 14]
```diff
@@ -0,0 +1,515 @@
...
+
+We use the package {% tool [RIAssigner](toolshed.g2.bx.psu.edu/repos/recetox/riassigner/riassigner/0.3.2+galaxy1) %} to compute retention indices for files in the `.msp` format using an indexed reference list of alkanes in `.csv` or `.msp` format. The output follows the same format as the input but with added retention index values. These can be used at a subsequent stage to improve compound identification ({% cite kumari2011applying %}). Multiple computation methods (e.g. piecewise-linear or cubic spline) are supported by the tool.
+
+> <hands-on-title> Retention index calculation </hands-on-title>
+>
+> 1. {% tool [RIAssigner](toolshed.g2.bx.psu.edu/repos/recetox/riassigner/riassigner/0.3.2+galaxy1) %} with the following parameters:
```
Comment (melpetera) [inline topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:386]:
```suggestion
> 1. {% tool [RIAssigner](toolshed.g2.bx.psu.edu/repos/recetox/riassigner/riassigner/0.3.4+galaxy1) %} with the following parameters:
```

File: topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga
[the remark this answers is not in the document]
Comment (melpetera) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/workflows/main_workflow.ga:1]: Ok. I added a suggestion to also change the version that appears in the tutorial.md file accordingly

**Comment by melpetera** [comment]
I know it may seems that I split hairs here, but to note this is because I think this tutorial is great, so I can't help suggesting what I think could make it even better :flushed:

File: topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (xtrojak) [reply topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md:157]: That's an excellent idea! I moved the "Skip" part to the suggested location in [7e2b8c4](https://github.com/galaxyproject/training-material/pull/3949/commits/7e2b8c4de1830b4537a90de1ed1636c86a9e2b24). I think you are right that this way, the trainee does not skip some important reading. Thank you for the detailed description.

**Comment by xtrojak** [comment]
> I know it may seems that I split hairs here, but to note this is because I think this tutorial is great, so I can't help suggesting what I think could make it even better flushed

@melpetera I tried to address your last comments. We are very grateful for your feedback and suggestions, they are very valuable!

**Comment by hechth** [comment]
@melpetera we added the already processed XCMS dataset to the Zenodo record and added a section stating that this can be used to skip the pre-processing part - like this we can keep things open if we want to add some specific about GC peak picking but also users have the option to start from the other tutorial or skip this section entirely.

The version for RIAssigner has been updated, so I think this should be all good now :).

**Review by melpetera — APPROVED** [review]
As discussed during previous review, ok for me :)

**Comment by melpetera** [comment]
Thank you for your work and the time spent on my suggestions!
All good for me :)

@hexylena Am I supposed to hit the "workflow" button now?

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
done so now. looks good! if it passes we merge :)

**Comment by hechth** [comment]
@melpetera and @hexylena thank you so much for the support! :)
